src/entity.js

```diff
@@ -0,0 +1,73 @@
+function runTransforms(transforms, entry) {
+  const [originalKey, originalValue] = entry
+
+  const transform = transforms.find(({ test }) =>
```

Comment:
I did it intentionally. The first transform that matched is chosen. If we apply more transforms, we increase complexity exponentially.

A transform can change both value and key. If the key changes, we should start from the beginning of the list. Also, initial state can have only numbers and strings but after transform it may become a class instance so transforms should expect that.

Reply:
To extend, if we have for example `affiliationUrl` and `dataProvidersUrl`, the first one becomes an Entity, another one is a DataProvidersCollection but both can match simple `/Url$/` test.

Reply:
So, higher priority goes first. We document it and follow it.